.idea/codeStyles/Project.xml

```diff
@@ -5,6 +5,22 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+        <value>
```

Comment:
@MaskedCarrot You should not add any changes to this file. Once you fix this, your PR will pass all the required tests.

Reply:
Yeah that's correct